Source/Examples/WPF/WpfExamples/Examples/AnimationsDemo/AnimationExtensions.cs

```diff
@@ -0,0 +1,112 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="MainViewModel.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace AnimationsDemo
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using System.Windows.Media.Animation;
```

Comment:
could this code be made portable? it would be interesting to see it in the portable example library...